server/controllers/userControlers.js

```diff
@@ -0,0 +1,57 @@
+
+const account = require('../models/userCreate')
+
+const userInfo = {
+    userSignUp(req, res) {
+        if ( account == 'undefined') {
+            return res.status(403).json({
+                "status":"error",
+                "error":"undefined mfck"
```

Comment:
You should return more descriptive error messages. `undefined mfck` does not tell the user is not clear enough. 
Also, ensure to return the appropriate HTTP error messages. Here is a link to help you with that: https://www.restapitutorial.com/httpstatuscodes.html